### PR TITLE
[12.0][MIG] rma_sale_analytic

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: rma_sale_analytic/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+{
+    "name": "Analytic Account in RMA sale",
+    "version": "10.0.1.0.0",
+    "author": "Eficent,"
+              "Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "website": "http://www.eficent.com",
+    "category": "Analytic",
+    "depends": ["rma_account", "rma_analytic"],
```
Comment (MiquelRForgeFlow) [inline rma_sale_analytic/__manifest__.py]: shouldn't this module depend of `rma_sale`? 🤔

Comment (AaronHForgeFlow) [reply rma_sale_analytic/__manifest__.py]: Yes

Comment (MiquelRForgeFlow) [reply rma_sale_analytic/__manifest__.py]: then, shouldn't this have `auto_install=True`?

Comment (AaronHForgeFlow) [reply rma_sale_analytic/__manifest__.py]: yes

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

**Pull request merged**